src/praisonai-agents/tests/managed/test_managed_trace_events.py

```
@@ -0,0 +1,239 @@
+        
+        # Mock session idle event
+        mock_idle = Mock()
+        mock_idle.type = "session.status_idle"
+        
+        # Set up trace sink
+        sink = ContextListSink()
+        emitter = ContextTraceEmitter(sink=sink, session_id="test_session", enabled=True)
+        
+        # Call _process_events with emitter
+        with trace_context(emitter):
+            text_parts, tool_log = agent._process_events(
```

Ruff (0.15.10)

[warning] Unpacked variable `text_parts` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)